src/utils.js

```diff
@@ -226,6 +226,57 @@ class Utils {
 
     return urlA.host === urlB.host;
   }
+
+  logWarning(message) {
+    console.error('[WARNING] Stormpath SDK: ' + message);
```

Comment:
I think that [`console.warn()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/warn) is more appropriate.